client/my-sites/upgrades/verify-email-forward/verify-email-forward.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External Dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal Dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+import SectionHeader from 'components/section-header';
+import Card from 'components/card';
+import FormButton from 'components/forms/form-button';
+
+class VerifyEmailForward extends React.Component {
+	render() {
+		const { translate, mailbox, destination } = this.props;
+		return (
+			<Main wideLayout>
+				<DocumentHead title={ translate( 'Verify Email Forwarding', { context: 'Header' } ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Verify your email forwarding address' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/email-fwd-confirm)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.